lib/paper_trail/record_trail.rb

```diff
@@ -502,7 +502,7 @@ def habtm_assoc_ids(habtm_assoc)
     end
 
     def log_version_errors(version, action)
-      version.logger.warn(
+      version.logger&.warn(
```

Comment:
OK that is certainly a fair critique.  I can resubmit with the conditional inlined explicitly.